analytic_engine/src/memtable/mod.rs

```diff
@@ -63,7 +67,54 @@ impl ToString for MemtableType {
     fn to_string(&self) -> String {
         match self {
             MemtableType::SkipList => MEMTABLE_TYPE_SKIPLIST.to_string(),
-            MemtableType::Columnar => MEMTABLE_TYPE_COLUMNAR.to_string(),
+            MemtableType::Column => MEMTABLE_TYPE_COLUMNAR.to_string(),
+        }
+    }
+}
+
+/// Layered memtable options
+/// If `mutable_segment_switch_threshold` is set zero, layered memtable will be
+/// disable.
+#[derive(Debug, Clone, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+pub struct LayeredMemtableOptions {
+    pub mutable_segment_switch_threshold: ReadableSize,
```

Comment:
Shall we declare it as usize? It seems there is no need to use ReadableSize here?

Reply:
We will also set it from config, `ReadableSize` may be preferable in config situation?